### PR TITLE
modify pitr doc for comments about log-backup.enable

### PR DESCRIPTION
**Pull request by joccau**
Signed-off-by: joccau <zak.zhao@pingcap.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: br/br-log-command-line.md [hunk 1]
```diff
@@ -9,11 +9,17 @@ summary: 了解如何使用 br log 命令行工具进行日志备份并恢复日
...
 ## 前置条件
 
+### 安装 BR
+
 使用日志备份前，你需要安装 BR。你可以选择以下任一方式安装 BR：
 
 * [使用 TiUP 在线安装](/migration-tools.md#使用-tiup-快速安装)（推荐）
 * [下载 TiDB 离线包](/download-ecosystem-tools.md)
 
+### 打开日志备份开关
```
Comment (ran-huang) [inline br/br-log-command-line.md:19]:
```suggestion
### 启用日志备份
```

**Review by lonng — APPROVED** [review]

File: br/br-log-command-line.md [hunk 2]
```diff
@@ -9,11 +9,17 @@ summary: 了解如何使用 br log 命令行工具进行日志备份并恢复日
...
+
+使用日志备份功能，需将 `TiKV` 端的日志备份开关参数 `log-backup.enable` 设为 `true`，修改配置参数的方法请参考 [修改配置参数](/maintain-tidb-using-tiup.md#修改配置参数)。
```
Comment (ran-huang) [inline br/br-log-command-line.md:21]:
```suggestion
使用日志备份功能前，需将 TiKV 配置项 [`log-backup.enable`](/tikv-configuration-file.md#enable-从-v620-版本开始引入) 设为 `true`。修改配置参数的方法，请参考[修改配置参数](/maintain-tidb-using-tiup.md#修改配置参数)。
```

File: br/pitr-usage.md [hunk 3]
```diff
@@ -41,6 +41,10 @@ summary: 了解如何使用 PiTR。
 - 安装：执行命令 `tiup install br:v6.2.0`
 - 升级：执行命令 `tiup update br:v6.2.0`
 
+## 开启 `TiKV` 端日志备份开关
+
+使用日志备份功能，需将 `TiKV` 端的日志备份开关参数 `log-backup.enable` 设置为 `true`，修改配置参数的方法请参考 [修改配置参数](/maintain-tidb-using-tiup.md#修改配置参数)。
```
Comment (ran-huang) [inline br/pitr-usage.md:46]: 同上

Comment (joccau) [inline br/pitr-usage.md:46]:
```suggestion
使用日志备份功能前，需将 TiKV 配置项 [`log-backup.enable`](/tikv-configuration-file.md#enable-从-v620-版本开始引入) 设为 `true`。修改配置参数的方法，请参考[修改配置参数](/maintain-tidb-using-tiup.md#修改配置参数)。
```

File: br/pitr-usage.md [hunk 4]
```diff
@@ -41,6 +41,10 @@ summary: 了解如何使用 PiTR。
 - 安装：执行命令 `tiup install br:v6.2.0`
 - 升级：执行命令 `tiup update br:v6.2.0`
 
+## 开启 `TiKV` 端日志备份开关
```
Comment (shichun-0415) [inline br/pitr-usage.md:44]:
```suggestion
## 启用日志备份
```

**Review by shichun-0415 — APPROVED** [review]
Rest LGTM

**Review by ran-huang — APPROVED** [review]

**Pull request merged**